### PR TITLE
Add argument 'origins' to Cors docs

### PR DESCRIPTION
**Pull request by carlosm27**
**Description**
When allowing CORS, if the is not specify, it will throw this error message:
image

This PR fixes #

### CODE REVIEW

**Comment by sansyrox** [comment]
Hey @carlosm27 👋 

Thank you for the PR 😄 I just saw this. Is it up for review?

**Comment by carlosm27** [comment]
> Hey @carlosm27 👋
> 
> Thank you for the PR 😄 I just saw this. Is it up for review?

Hey @sansyrox, don't worry. Yes.

File: docs_src/src/pages/documentation/api_reference/cors.mdx [hunk 1]
```diff
@@ -21,14 +21,14 @@ You can allow CORS for your application by adding the following code:
       from robyn import Robyn, ALLOW_CORS
 
       app = Robyn(__file__)
-      ALLOW_CORS(app)
+      ALLOW_CORS(app, origins = "http://localhost:<PORT>/")
```
Comment (sansyrox) [inline docs_src/src/pages/documentation/api_reference/cors.mdx:24]: Hey @carlosm27 👋 

Thank you for your PRs ✨ but origins is a `list[str]` and not just a `str`

**Review by sansyrox — APPROVED** [review]
LGTM! Thank you @carlosm27 😄

**Pull request merged**